internal/config.go

```diff
@@ -123,3 +126,8 @@ func MakeConfig(cmd *cobra.Command) (*hazelcast.Config, error) {
 	}
 	return config, nil
 }
+
+func DefaultConfigFile() string {
+	homeDirectoryPath, _ := os.UserHomeDir()
```

Comment:
How about renaming that to `home`?

Reply:
Also, the error should not be ignored. Ideally we should return the error, but if you think it is hard to integrate with the rest of the code you can also panic with something like:
```
panic(fmt.Error("retrieving home directory: %w", err))
```

Reply:
When I examined os.UserHomeDir, I saw that it only returns error if it is undefined, which very unlikely in any modern system. Returning error will make it difficult to use this function. If we must, I think panic is a better choice for this